features/proxy-fields.feature

```diff
@@ -0,0 +1,225 @@
+# language: nl
+
+
+Functionaliteit: vertalen van gevraagde samengestelde of afgeleide velden naar vragen van bron velden
+  De proxy stelt sommige velden samen uit een of meerdere andere velden. Hiervoor moet de fields in het request worden vertaald van het doelveld naar de verschillende bronvelden.
+
+  Rule: wanneer in het request een afgeleid of samengesteld veld wordt gevraagd, moet deze in het request naar de RvIG API worden vervangen door de velden waaruit deze wordt samengesteld
+    Het gaat om de volgende vertalingen:
+      | gevraagd aan proxy                 | vragen aan RvIG                                                                                           |
```

Comment:
Is het niet beter om dit lijstje toe te voegen aan onderstaand abstract scenario? Als het wordt toegevoegd als documentatie, dan kan je het net zo goed toevoegen aan de voorbeelden

Reply:
dus niet eerst specificeren hoe het moet werken, maar alleen via voorbeeld laten zien?

Reply:
ik heb het lijstje direct onder de rule verwijderd. Deze zitten nu in de voorbeelden

Reply:
Vermoed ik ook, want ik had het wel gewijzigd. Zie dat de wijziging nu is doorgekomen